portfolio/src/components/Home.vue

```diff
@@ -89,18 +91,8 @@ export default {
 
 <!-- Add "scoped" attribute to limit CSS to this component only -->
 <style scoped>
-    #home-container {
-        height: 100%;
-        width: 100%;
-
-        display: flex;
-        flex-direction: column;
-        align-items: center;
-        justify-content: flex-start;
-
-        padding: 100px 16px 16px 16px;
-
-        background-color: transparent;
+    #home-container {        
```

Comment:
remove?

Reply:
Ahh yeah the remnants of the previous  responsiveness approach, will do!